crates/sui-benchmark/src/workloads/payload.rs

```diff
@@ -1,15 +1,16 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+use crate::workloads::workload::WorkloadType;
 use crate::ExecutionEffects;
-use std::fmt::Display;
 use sui_types::messages::VerifiedTransaction;
 
 /// A Payload is a transaction wrapper of a particular type (transfer object, shared counter, etc).
 /// Calling `make_transaction()` on a payload produces the transaction it is wrapping. Once that
 /// transaction is returned with effects (by quorum driver), a new payload can be generated with that
 /// effect by invoking `make_new_payload(effects)`
-pub trait Payload: Send + Sync + std::fmt::Debug + Display {
+pub trait Payload: Send + Sync {
     fn make_new_payload(&mut self, effects: &ExecutionEffects);
     fn make_transaction(&mut self) -> VerifiedTransaction;
+    fn workload_type(&self) -> WorkloadType;
```

Comment:
Payload already has a display type, can we just use the string instead.I think that keep things simple. A big goal for me with `sui-benchmarking` framework code is to keep it really simple for folks to be able to understand and add to it (after being told many times that it is complicated)